### PR TITLE
Remove the small image

### PR DESCRIPTION
**Pull request by DivyanshVinayak23**

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Please add the center to the image only bcs if you put it to the download btn a chance of another will missed out @DivyanshVinayak23

File: index.html [hunk 1]
```diff
@@ -66,16 +66,16 @@
...
         <img src="assets/img/CALCULATOR.png">
-    </center>
+    
+        <br>
+        <br>
         
-
    <a href="Application/CALCULATOR.exe" class="btn btn-success btn-sm"><i class="fa-solid fa-download"></i> Download</a>
-  
+</center>
```
Comment (YoshithaRathnayake) [inline index.html:78]: Please add the center to the image only bcs if you put it to the download btn a chance of another will missed out @DivyanshVinayak23

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Resolve the conflicts also @DivyanshVinayak23

**Comment by DivyanshVinayak23** [comment]
Done I have fixed it

**Comment by YoshithaRathnayake** [comment]
Resolve the conflicts also @DivyanshVinayak23

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You!
But the resolving part I did that's very very bad @DivyanshVinayak23

**Pull request merged**